### DISCUSSION TITLE
设置字体后只在本地hexo运行显示正确，但是部署到github就不行了。

### DISCUSSION THREAD

**Discussion by yeelvus** [xaoxuu/hexo-theme-stellar#271] [Q&A]
使用了介绍的导入外部字体的方式，也使用了系统字体配置。均是在本地hexo运行有效果，上传到GitHub部署后就没有效果了，不知道是配置的问题还是网页加载外部字体失败了。
![image](https://user-images.githubusercontent.com/50106230/219847131-69d01107-fb5d-4185-9a06-be4a5b7a7e11.png)
![image](https://user-images.githubusercontent.com/50106230/219847145-9d1590ad-47b6-4970-bd20-b34bebbfcc03.png)
![image](https://user-images.githubusercontent.com/50106230/219847177-f9f1f341-dd61-4b1c-9061-7cac51d24edb.png)
使用hexo s的本地部署后在网页显示为：
font-family: "Noto Serif SC", "LXGW WenKai Screen", system-ui, "Microsoft Yahei", "Segoe UI", -apple-system, Roboto, Ubuntu, "Helvetica Neue", Arial, "WenQuanYi Micro Hei", sans-serif;
![image](https://user-images.githubusercontent.com/50106230/219847358-98fb6349-412f-4831-a485-b8215cfec77a.png)
部署到GitHub后显示为：font-family: system-ui, "Microsoft Yahei", "Segoe UI", -apple-system, Roboto, Ubuntu, "Helvetica Neue", Arial, "WenQuanYi Micro Hei", sans-serif;
![image](https://user-images.githubusercontent.com/50106230/219847384-7dbf0892-a42a-4fd0-a026-4303d08d9329.png)

**Comment by yeelvus** [comment]
还请大佬解答，主题非常喜欢😀。

**Comment by xaoxuu** [comment]
感觉像是有缓存

**Comment by Ashelyjiang** [comment]
我也是一样的问题，请问解决了嘛~

**Comment by yeelvus** [comment]
已经解决了。 https://lvmogu.xyz/2024/01/16471/  我刚好做了记录。抱歉忘记回复了

Ashelyjiang ***@***.***> 于2024年9月15日周日 17:10写道：
